static/app/components/replays/breadcrumbs/timelineGaps.tsx

```diff
@@ -16,13 +16,14 @@ interface Props {
 
 export default function TimelineGaps({durationMs, startTimestampMs, videoEvents}: Props) {
   const organization = useOrganization();
+  const sortedVideoEvents = videoEvents.sort((a, b) => a.timestamp - b.timestamp);
```

Comment:
We should make sure the video events are always sorted (i.e. let's do this higher up in the component tree so that it's sorted for all components)